### ISSUE TITLE
2x missing dependencies

### ISSUE THREAD

**Issue by SimeonGriggs** [kaicataldo/eleventy-plugin-react#4]
Hey, just trying this out on a fresh 11ty instance and got these warning trying the first build. 

```
> Cannot find module '@babel/preset-react' from '/Users/home/Sites/react-11ty'
> Cannot find module '@babel/preset-env' from '/Users/home/Sites/react-11ty'
```

After installing them, first build worked fine. Do they need to be added to the list of dependencies in the readme?

**Comment by dpfavand** [comment]
I ran into this as well when first installing. Suggesting a readme update in #5

**Comment by kaicataldo** [comment]
Yeah, this was something I was unsure about. I initially wrote this with all the Babel configuration hardcoded in the plugin (treating Babel as a direct dep rather than a peer dep), because I'm generally all for sane defaults and I know there's a lot of friction to having to set up all this tooling. I then had second thoughts about this, because I figured there would be end users who would _want_ to configure Babel (beyond supplying targets), for instance if they wanted to use TypeScript or had some custom plugins (Emotion comes to mind), and ended up here.

Listing those plugins as a peer deps sounds good to me in theory - my biggest concern with that was that I wanted to leave end users to have the freedom to configure Babel as they see fit. Maybe there's a middle ground here - sane defaults (with a hardcoded underlying config with direct deps), with an API that allows you to modify the config somehow?

**Comment by kaicataldo** [comment]
Thanks for bringing this to my attention again - definitely doesn't seem like a good user experience! Let's continue this discussion over on https://github.com/kaicataldo/eleventy-plugin-react/pull/5. I'm leaning towards making them peer deps for now and then potentially looking into my suggestion above in the future.
